Sources/AblyChat/Errors.swift

```diff
@@ -11,10 +11,8 @@ public let errorDomain = "AblyChatErrorDomain"
  The error codes for errors in the ``errorDomain`` error domain.
  */
 public enum ErrorCode: Int {
-    /// ``Rooms.get(roomID:options:)`` was called with a different set of room options than was used on a previous call. You must first release the existing room instance using ``Rooms.release(roomID:)``.
-    ///
-    /// TODO this code is a guess, revisit in https://github.com/ably-labs/ably-chat-swift/issues/32
-    case inconsistentRoomOptions = 1
+    /// The user attempted to perform an invalid action.
+    case nonspecificUserError = 40000
```

Comment:
Should it be `badRequest` - which matches up with [errors.json](https://github.com/ably/ably-common/blob/main/protocol/errors.json)?

Reply:
Good shout — will change

Reply:
Done!

Reply:
Thanks! 🎉